tools/unicode_properties_parse/.gitignore

```diff
@@ -0,0 +1,14 @@
+#// clang-format off

```

Comment:
Should we instead teach [`parallelize.cpp`](https://github.com/microsoft/STL/blob/main/tools/parallelize/parallelize.cpp) to stop treating `.gitignore` as a source file?

Reply:
Agreed that we should exclude `.gitignore` from clang-format.